client/asset/eth/eth.go

```diff
@@ -2798,18 +2798,14 @@ func (*baseWallet) ValidateSecret(secret, secretHash []byte) bool {
 // more, requesting the best block header starts to fail after a few tries
 // during initial sync. Investigate how to get correct sync progress.
 func (eth *baseWallet) SyncStatus() (bool, float32, error) {
-	prog, err := eth.node.syncProgress(eth.ctx)
+	prog, bestHeaderUNIXTime, err := eth.node.syncProgress(eth.ctx)
 	if err != nil {
 		return false, 0, err
 	}
 	checkHeaderTime := func() (bool, error) {
-		bh, err := eth.node.bestHeader(eth.ctx)
-		if err != nil {
-			return false, err
-		}
 		// Time in the header is in seconds.
-		timeDiff := time.Now().Unix() - int64(bh.Time)
-		if timeDiff > dexeth.MaxBlockInterval && eth.net != dex.Simnet {
```

Comment:
Removing the simnet check and mining some blocks in the harness tests. It seems reasonable to just mine some blocks when testing. May be a preference thing so I can revert.

Reply:
Well, ok. Also passing the net to prividers `bestHeader` so it can also not care.